### ISSUE TITLE
Using get_post_meta() for custom order meta in Email templates

### ISSUE THREAD

**Issue by rnevius** [woothemes/woocommerce#6982]
Related [post on StackOverflow](http://stackoverflow.com/questions/25635461/get-post-meta-in-woocommerce-email-notifications).

TL;DR: [get_post_meta()](http://codex.wordpress.org/Function_Reference/get_post_meta) doesn't seem to work within email templates, when used to get custom meta values associated with the order. WooCommerce 2.2.10, WordPress 4.1.

For example, if we add the following to **functions.php** to create a new order meta item during checkout:

    /**
     * Add "pin" field to the checkout.
     */
    function pin_field( $checkout ) {
        echo '<div id="pin_field">';
     
        woocommerce_form_field( 'pin', array(
            'type'          => 'text',
            ), $checkout->get_value( 'pin' ));
     
        echo '</div>';
    }
    add_action( 'woocommerce_after_order_notes', 'pin_field' );

    /**
     * Update the order meta with pin_field value
     */
    function pin_field_update_order_meta( $order_id ) {
        if ( ! empty( $_POST['pin'] ) ) {
            update_post_meta( $order_id, '_pin', sanitize_text_field( $_POST['pin'] ) );
        }
    }
    add_action( 'woocommerce_checkout_update_order_meta', 'pin_field_update_order_meta' );

...the order meta saves to the database just fine. However, if we then try to call it in a template such as **admin-new-order.php** using `<?php echo get_post_meta( $order->id, '_pin', true ); ?>`, nothing is returned.

`<?php echo get_post_meta( $order->id, '_pin', true ); ?>` *does* work elsewhere to retrieve the meta info (after the order is complete and emails are sent). It just doesn't seem to retrieve the appropriate data during the email process.

**Comment by claudiosmweb** [comment]
I tested it and everything is working great (or at least it will be in 2.3).

**Comment by mikejolley** [comment]
I checked too; new order is triggered after meta is saved in 2.3

**Comment by wielsma** [comment]
I'm experiencing the same problem. When testing the e-mail template using the built in 'resend e-mail' function on the order itself in /wp-admin/ it works fine (can read the meta data) but when I change the status of the order to 'completed' from within the orders index, the template is unable to get the meta_data. I've tried to embed the function right inside the e-mail template, but stil nothing is returned. 

Strange, strange...
